source/modules/MEI.cmn.xml

```diff
@@ -2262,6 +2262,53 @@
         model.textPhraseLike.limited.</p>
     </remarks>
   </elementSpec>
+  <elementSpec ident="repeatMark" module="MEI.cmn">
```

Comment:
'coda' and 'fine' are not really _repeat marks_ (even though they often appear in repetition contexts). Most of the presented values here mark jumps, so I'd prefer something like `jumpMark`.

Reply:
Yes, you got it right. The element represents marks within repetition patterns. It is a matter of preference.

Reply:
`jumpMark` sounds too "jargon-y" to me, too "modern" perhaps.  These are "navigation marks" to me, so "navMark" is a possibility.  Still, I prefer "repeatMark".

Reply:
`<sequenceMark>`
`<sequencingMark>`
`<seqMark>` (short form of either of the above)
`<execMark>` (short for execution mark)
`<expanMark>` (short for expansion mark, related to `<expansion>`)

I think I like the last one best because it indicates the function of the mark is analogous to the functionality of `<expansion>`.

Reply:
`performMark`?

Reply:
I know `<expanMark>` is somewhat cryptic, but I prefer it over `<performMark>`.  `<performMark>` creates semantic confusion between "performance marks", like coda, segno, etc. and "performance directives"; i.e., `<dir>` elements. It's the same reason `<repeatMark>` isn't a very good name -- it can be confused with repeat marks on barlines.  At least, `<expanMark>` aligns semantically with `<expansion>`, the description of which (I hope) makes the concept clearer.

Reply:
This all makes sense. My only concern with `<expanMark>` is similar to yours with `<performMark>`, namely that it could create confusion, with `<expansion>`. In the end users will need to read the docs, and even XML is self-documenting, we still want to avoid names becoming too long in order just because we want to make sure there is no possible confusion. So I am still fine with `<repeatMark>`. (I am fine with changing it too if we feel it is seriously wrong or somebody finds something much better.)